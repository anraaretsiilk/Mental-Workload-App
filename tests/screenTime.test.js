import React from 'react';
import renderer from 'react-test-renderer';
import ScreenTime from '../screens/screenTime';

test('renders screen time correctly', () => {
  const tree = renderer.create(<ScreenTime />).toJSON();
  expect(tree).toMatchSnapshot();
});