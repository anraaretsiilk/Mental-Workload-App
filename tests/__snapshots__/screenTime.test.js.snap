// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders screen time correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 20,
    }
  }
>
  <Text>
    Screen Time
  </Text>
</View>
`;
